test/jdk/java/awt/List/MouseDraggedOriginatedByScrollBarTest.java

```diff
@@ -24,41 +24,46 @@
 /*
  * @test
  * @bug 6240151
+ * @key headful
  * @summary XToolkit: Dragging the List scrollbar initiates DnD
- * @library /java/awt/regtesthelpers
- * @build PassFailJFrame
- * @run main/manual MouseDraggedOriginatedByScrollBarTest
+ * @requires os.family == "linux"
+ * @run main MouseDraggedOriginatedByScrollBarTest
 */
 
+import java.awt.EventQueue;
 import java.awt.FlowLayout;
 import java.awt.Frame;
 import java.awt.List;
-import java.awt.event.MouseMotionAdapter;
+import java.awt.Point;
+import java.awt.Robot;
+import java.awt.event.InputEvent;
 import java.awt.event.MouseAdapter;
 import java.awt.event.MouseEvent;
+import java.awt.event.MouseMotionAdapter;
 
 public class MouseDraggedOriginatedByScrollBarTest {
-
-    private static final String INSTRUCTIONS = """
-            1) Click and drag the scrollbar of the list.
-            2) Keep dragging till the mouse pointer goes out the scrollbar.
-            3) The test failed if you see messages about events. The test passed if you don't.""";
+    private static Frame frame;
+    private static volatile Point loc;
+    private static volatile List list;
```

Comment:
```suggestion
    private static List list;
```
There's no reason for `list` to be volatile.